### ISSUE TITLE
self-signed certificate error behind traffic intercepting proxy

### ISSUE THREAD

**Issue by mrhorvath** [microsoft/vscode-pull-request-github#4807]
- Extension version: v0.62.0
- VSCode Version: 1.78.0
- OS: Microsoft Windows 10 Enterprise

Steps to Reproduce:

1. Be behind proxy intercepting traffic with self-signed certificate
2. Refresh Pull Requests List

I cannot view open pull requests from the extension.  The output window for Github Pull Request shows:

```
2023-05-04 14:46:15.753 [error] GitHubRepository> Fetching pull requests for branch failed: Error: Network error: request to https://api.github.com/graphql failed, reason: self signed certificate in certificate chain
```

My machine resides behind a corporate firewall that intercepts traffic, but the windows store contains the certificates necessary to establish trust.  Does the extension not load the windows trust store by default, or is there a way to configure the extension to have it use the windows trust store?

Note that I can confirm that the certificates in the windows store are functioning as expected (opening api.github.com shows a secure connection using the corporate certificate)

**Comment by ba-work** [comment]
same here, seems to be related to 1.78.

i downgraded to 1.77.3 and it started working again

**Comment by alexr00** [comment]
Since the issue is fixed by downgrading to 1.77 it's likely something changed in VS Code. I'm following up to see what that might be.

**Comment by alexr00** [comment]
This looks like it's related to https://github.com/microsoft/vscode/issues/181404. The fix is available in 1.78.2.
